.github/workflows/examples.yml

```diff
@@ -45,3 +45,9 @@ jobs:
         run: pip install nox
       - name: Run examples
         run: nox -s examples
+      - name: Generate HTML
+        run: nox -s examples -- html
+      - uses: actions/upload-artifact@v4
+        with:
+          name: Examples
+          path: examples/**/*.html
```

Comment:
I think this now executes all of the examples twice: once with `jupyter execute examples/**/*.ipynb` and once with `jupyter nbconvert --execute`. You could either remove the first step, or I believe you can remove `--execute` from the second step.